site/data/plugins.yml

```diff
@@ -0,0 +1,47 @@
+- name: Alert
```

Comment:
Couldn't we get this info via a front matter property and the page description? Just wondering if we really need a new file, not against it per se :)

Reply:
Open to that as an optimization in the future. For now, keeping this as-is.